README.md

```diff
@@ -29,9 +29,9 @@ security of the system, such as auditing and explainability features.
 
 ## Members
 
-* Dan Shaw ([@dshaw](https://github.com/dshaw)), PayPal [chair]
-* Sarah Allen ([@ultrasaurus](https://github.com/ultrasaurus)), [chair]
-* Jeyappragash JJ ([@pragashj](https://github.com/pragashj)), Tetrate.io [chair]
+* Dan Shaw ([@dshaw](https://github.com/dshaw)), PayPal [Chair - term: 1 year - start date: 6/3/2019 - 6/3/2020]
+* Sarah Allen ([@ultrasaurus](https://github.com/ultrasaurus)), [Chair - term: 2 years - start date: 6/3/2019 - 6/3/2021]
```

Comment:
nit: I would remove " - start date" since the text following that is a range

Also, I would be inclined to simplify and just have format as [Chair 6/3/2019 - 6/3/2021]

Reply:
Let's merge this and when i rebase https://github.com/cncf/sig-security/pull/280 I will make this change?